scanner/src/test/java/edu/ucr/cs/riple/scanner/tools/MethodInfoDisplay.java

```diff
@@ -46,6 +47,7 @@ public MethodInfoDisplay(
       String size,
       String flags,
       String hasNullableAnnotation,
+      String visibility,
```

Comment:
This `display` classes are just instances to help us read serialized outputs, therefore they need contain all the serialized information on the output including the `visibility` to run the tests successfully.

Reply:
Maybe my suggestion didn't come across. What I was suggesting is `MethodInfoDisplay(String ... args)` because in method info test, you have to awkwardly parse the array. But it's fine, it's just a nit.  Since I cant actually comment upon the functionality, I am acting as a human linter. :P